Makefile

```diff
@@ -9,10 +9,11 @@ EXAMPLES_DEPS = \
 		File::Temp \
 		Text::VimColour \
 		HTTP::Easy \
-		Web::Request \
 		Term::ANSIColor \
 		Term::termios
 
+#		Web::Request \
```

Comment:
If `Web::Request` isn't needed, it can simply be deleted rather than being commented out.